src/search.cpp

```diff
@@ -782,7 +782,8 @@ namespace {
         &&  ss->staticEval >= beta - 21 * depth - improvement / 13 + 258
         && !excludedMove
         &&  pos.non_pawn_material(us)
-        && (ss->ply >= thisThread->nmpMinPly))
+        && (ss->ply >= thisThread->nmpMinPly)
```

Comment:
The parentheses around the expression can be removed.
```suggestion
        &&  ss->ply >= thisThread->nmpMinPly
```